public/img/logo.svg

```diff
@@ -0,0 +1,246 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I don't think that the Red Hat logo should be in the upstream project at all since this can be added by RHMI. @wei-lee - is this correct?

Reply:
I removed the logo.  If it needs to be in the repo then we can add it back.